src/com/lftechnology/java/training/sanish/stemleaf/UserInput.java

```diff
@@ -0,0 +1,50 @@
+package com.lftechnology.java.training.sanish.stemleaf;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Provide methods to get inputs form console
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class UserInput {
+	private final static Logger LOGGER = Logger.getLogger(UserInput.class
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)